src/config_map.c

```diff
@@ -107,8 +107,9 @@ const CONFIG_OPTION g_ConfigOptionMap[] = {
     { .name = "walk_to_items",                  .type = COT_BOOL,   .target = &g_Config.walk_to_items,                       .default_value = &(bool){false},                      0},
     { .name = "disable_trex_collision",         .type = COT_BOOL,   .target = &g_Config.disable_trex_collision,              .default_value = &(bool){false},                      0},
     { .name = "start_lara_hitpoints",           .type = COT_INT32,  .target = &g_Config.start_lara_hitpoints,                .default_value = &(int32_t){LARA_HITPOINTS},          0},
-    { .name = "healthbar_showing_mode",         .type = COT_ENUM,   .target = &g_Config.healthbar_showing_mode,              .default_value = &(int32_t){BSM_FLASHING_OR_DEFAULT}, .param = m_BarShowingModes},
-    { .name = "airbar_showing_mode",            .type = COT_ENUM,   .target = &g_Config.airbar_showing_mode,                 .default_value = &(int32_t){BSM_DEFAULT},             .param = m_BarShowingModes},
+    { .name = "healthbar_showing_mode",         .type = COT_ENUM,   .target = &g_Config.healthbar_show_mode,                 .default_value = &(int32_t){BSM_FLASHING_OR_DEFAULT}, .param = m_BarShowModes},
```

Comment:
Thanks; made it to match.

Reply:
Unfortunately the lang keys no longer match from the first change 😄 This fixes it, I've built in VS and tested each language, everything loads in fine. Retaining `_showing_` after all is probably good as otherwise people's settings would reset 👍 

```diff
diff --git a/tools/config/TR1X_ConfigTool/Resources/Lang/en-GB.json b/tools/config/TR1X_ConfigTool/Resources/Lang/en-GB.json
index 95d2402b..bc00d50c 100644
--- a/tools/config/TR1X_ConfigTool/Resources/Lang/en-GB.json
+++ b/tools/config/TR1X_ConfigTool/Resources/Lang/en-GB.json
@@ -15,14 +15,14 @@
     "change_pierre_spawn": {
       "Title": "Change Pierre spawn behaviour"
     },
-    "healthbar_show_mode": {
+    "healthbar_showing_mode": {
       "Title": "Healthbar behaviour"
     },
     "healthbar_color": {
       "Title": "Healthbar colour",
       "Description": "Colour of the healthbar."
     },
-    "airbar_show_mode": {
+    "airbar_showing_mode": {
       "Title": "Airbar behaviour"
     },
     "airbar_color": {
diff --git a/tools/config/TR1X_ConfigTool/Resources/Lang/en.json b/tools/config/TR1X_ConfigTool/Resources/Lang/en.json
index 23aaf8bd..ca56d960 100644
--- a/tools/config/TR1X_ConfigTool/Resources/Lang/en.json
+++ b/tools/config/TR1X_ConfigTool/Resources/Lang/en.json
@@ -377,7 +377,7 @@
       "Title": "Menu style",
       "Description": "Changes how menus are displayed.\n - PC: UI style matches the PC version.\n - PS1: UI style matches the PS1 version."
     },
-    "healthbar_show_mode": {
+    "healthbar_showing_mode": {
       "Title": "Healthbar behavior",
       "Description": "Changes how the healthbar is displayed.\n- Default: show the healthbar at the start of a level, after getting hit, or while having weapons equipped; flash the healthbar if it's already on screen when Lara's health is 20% or below (TombATI).\n- Flashing or default: constantly flash the healthbar when Lara's health is 20% or below, or show under default circumstances.\n- Flashing only: show the healthbar *only* when Lara's health is 20% or below (for challenge runs).\n- Always: always show the healthbar.\n- Never: never display the healthbar (for challenge runs).\n- PS1: show the healthbar under default circumstances with no flashing."
     },
@@ -389,7 +389,7 @@
       "Title": "Healthbar color",
       "Description": "Color of the healthbar."
     },
-    "airbar_show_mode": {
+    "airbar_showing_mode": {
       "Title": "Airbar behavior",
       "Description": "Changes how the airbar is displayed.\n- Default: show the airbar only in the water and flash the airbar when Lara's oxygen is 20% or below (TombATI).\n- Flashing only: show the airbar *only* when Lara's oxygen is 20% or below (for challenge runs).\n- Never: never display the airbar (for challenge runs).\n- PS1: show the airbar under default circumstances with no flashing."
     },
@@ -401,8 +401,8 @@
       "Title": "Airbar color",
       "Description": "Color of the airbar."
     },
-    "enemy_healthbar_show_mode": {
-      "Title": "Show enemy Healthbars",
+    "enemy_healthbar_showing_mode": {
+      "Title": "Show enemy healthbars",
       "Description": "Enables showing a healthbar for the active enemy."
     },
     "enemy_healthbar_location": {
diff --git a/tools/config/TR1X_ConfigTool/Resources/Lang/es.json b/tools/config/TR1X_ConfigTool/Resources/Lang/es.json
index 2c4d1bc2..5732a836 100644
--- a/tools/config/TR1X_ConfigTool/Resources/Lang/es.json
+++ b/tools/config/TR1X_ConfigTool/Resources/Lang/es.json
@@ -101,7 +101,7 @@
       "Title": "Ubicación de la barra de aire",
       "Description": "Ubicación donde se muestra la barra de aire."
     },
-    "airbar_show_mode": {
+    "airbar_showing_mode": {
       "Title": "Modo de visualización de la barra de aire",
       "Description": "Cambia la forma en que se muestra la barra de aire.\n- Predeterminado: muestra la barra de aire solo cuando Lara está bajo el agua y parpadea cuando el oxígeno está al 20% o menos (TombATI).\n- Solo parpadeo: muestra la barra de aire *solo* cuando el oxígeno de Lara está al 20% o menos (para carreras de desafío).\n- Nunca: nunca muestra la barra de aire (para carreras de desafío).\n- PS1: muestra la barra de aire en circunstancias predeterminadas sin parpadeo."
     },
@@ -177,7 +177,7 @@
       "Title": "Mostrar total de objetos y muertes",
       "Description": "Permite mostrar el recuento máximo de objetos recogidos y el recuento de muertes en cada nivel. Esto incluye objetos que no se pueden obtener."
     },
-    "enemy_healthbar_show_mode": {
+    "enemy_healthbar_showing_mode": {
       "Title": "Mostrar barra de salud enemiga",
       "Description": "Permite mostrar una barra de salud para el enemigo activo."
     },
@@ -381,7 +381,7 @@
       "Title": "Ubicación de la barra de salud",
       "Description": "Ubicación donde se muestra la barra de salud."
     },
-    "healthbar_show_mode": {
+    "healthbar_showing_mode": {
       "Title": "Modo de visualización de la barra de salud",
       "Description": "Cambia cómo se muestra la barra de salud.\n- Predeterminado: muestra la barra de salud al comienzo de un nivel, después de recibir un golpe o mientras tiene armas equipadas; muestra la barra de salud si ya está en la pantalla cuando la salud de Lara es del 20% o menos (TombATI).\n- Intermitente o predeterminada: la barra de salud parpadea constantemente cuando la salud de Lara es del 20% o menos, o se muestra en circunstancias predeterminadas.\n- Solo intermitente: muestra la barra de salud *solo* cuando la salud de Lara es del 20% o menos (para carreras de desafío).\n- Siempre: muestra siempre la barra de salud.\n- Nunca: nunca mostrar la barra de salud (para carreras de desafío).\n- PS1: muestra la barra de salud en circunstancias predeterminadas sin parpadear."
     },
diff --git a/tools/config/TR1X_ConfigTool/Resources/Lang/fr.json b/tools/config/TR1X_ConfigTool/Resources/Lang/fr.json
index decca524..a8f310e8 100644
--- a/tools/config/TR1X_ConfigTool/Resources/Lang/fr.json
+++ b/tools/config/TR1X_ConfigTool/Resources/Lang/fr.json
@@ -377,7 +377,7 @@
       "Title": "Style des menus",
       "Description": "Changer le rendu des menus.\n - PC: Style d'interface de la version PC.\n - PS1: Style d'interface de la version PlayStation."
     },
-    "healthbar_show_mode": {
+    "healthbar_showing_mode": {
       "Title": "Comportement de la barre de santé",
       "Description": "Modifie l'affichage de la barre de santé.\n- Par défaut : affiche la barre de santé au début d'un niveau, après avoir été touché ou lorsque des armes sont équipées ; fait clignoter la barre de santé si elle est déjà à l'écran lorsque la santé de Lara est de 20 % ou moins (TombATI).\n- Clignotant ou par défaut : la barre de santé clignotte constamment lorsque la santé de Lara est à 20 % ou moins, ou s'affiche dans des circonstances par défaut.\n- Clignotant uniquement : affiche la barre de santé *uniquement* lorsque la santé de Lara est à 20 % ou moins (pour les courses de défi ).\n- Toujours : affiche toujours la barre de santé.\n- Jamais : n'affiche jamais la barre de santé (pour les défis).\n- PS1 : affiche la barre de santé dans les circonstances par défaut sans clignoter."
     },
@@ -389,7 +389,7 @@
       "Title": "Couleur de la barre de santé",
       "Description": "Choisir la couleur de la barre de santé de Lara."
     },
-    "airbar_show_mode": {
+    "airbar_showing_mode": {
       "Title": "Comportement de la barre d'air",
       "Description": "Modifie l'affichage de la barre d'air.\n- Par défaut : affiche la barre d'air uniquement dans l'eau et clignote lorsque l'oxygène de Lara est à 20 % ou moins (TombATI).\n- Clignotant uniquement : affiche la barre d'air *uniquement* lorsque l'oxygène de Lara est à 20 % ou moins (pour challenge runs).\n- Jamais : n'affiche jamais la barre d'air (pour les challenge runs).\n- PS1 : affiche la barre d'air dans les circonstances par défaut sans clignoter."
     },
@@ -401,7 +401,7 @@
       "Title": "Couleur de la barre d'air",
       "Description": "Choisir la couleur de la barre d'air."
     },
-    "enemy_healthbar_show_mode": {
+    "enemy_healthbar_showing_mode": {
       "Title": "Afficher la barre de santé des ennemis",
       "Description": "Affiche la barre de santé de l'ennemi sur lequel Lara vise et tire"
     },
diff --git a/tools/config/TR1X_ConfigTool/Resources/Lang/it.json b/tools/config/TR1X_ConfigTool/Resources/Lang/it.json
index f4fac7b1..38c0650f 100644
--- a/tools/config/TR1X_ConfigTool/Resources/Lang/it.json
+++ b/tools/config/TR1X_ConfigTool/Resources/Lang/it.json
@@ -377,7 +377,7 @@
       "Title": "Stile del menù",
       "Description": "Modifica la modalità di visualizzazione dei menu.\n - PC: Lo stile dell'interfaccia utente corrisponde alla versione PC.\n - PS1: Lo stile dell'interfaccia utente corrisponde alla versione PS1."
     },
-    "healthbar_show_mode": {
+    "healthbar_showing_mode": {
       "Title": "Comportamento della barra della salute",
       "Description": "Modifica la modalità di visualizzazione della barra della salute.\n- Predefinito: mostra la barra della salute all'inizio di un livello, dopo essere stato colpito o mentre si equipaggiano\n  le armi; se già sullo schermo, la barra della salute lampeggia quando la salute di Lara è pari o inferiore al 20%\n  (TombATI).\n- Lampeggiante o predefinito: la barra della salute lampeggia costantemente quando la salute di Lara è pari o\n  inferiore al 20%, oppure viene visualizzata in circostanze predefinite.\n- Solo lampeggiante: mostra la barra della salute *solo* quando la salute di Lara è pari o inferiore al 20%\n  (per le sfide).\n- Sempre: mostra sempre la barra della salute.\n- Mai: non visualizzare mai la barra della salute (per le sfide).\n- PS1: mostra la barra della salute in circostanze predefinite senza mai lampeggiare."
     },
@@ -389,7 +389,7 @@
       "Title": "Colore della barra della salute",
       "Description": "Colore utilizzato per la barra della salute."
     },
-    "airbar_show_mode": {
+    "airbar_showing_mode": {
       "Title": "Comportamento della barra dell'ossigeno",
       "Description": "Modifica la modalità di visualizzazione della barra dell'ossigeno.\n- Predefinito: mostra la barra dell'ossigeno solo in acqua e lampeggia quando l'ossigeno di Lara è pari o inferiore\n  al 20% (TombATI).\n- Solo lampeggiante: mostra la barra dell'ossigeno *solo* quando l'ossigeno di Lara è pari o inferiore al 20%\n  (per le sfide).\n- Mai: non visualizzare mai la barra dell'ossigeno (per le sfide).\n- PS1: mostra la barra dell'ossigeno in circostanze predefinite senza mai lampeggiare."
     },
@@ -401,7 +401,7 @@
       "Title": "Colore della barra dell'ossigeno",
       "Description": "Colore utilizzato per la barra dell'ossigeno."
     },
-    "enemy_healthbar_show_mode": {
+    "enemy_healthbar_showing_mode": {
       "Title": "Mostra la barra della salute dei nemici",
       "Description": "Abilita la visualizzazione della barra della salute per il nemico ingaggiato."
     },
```

Reply:
Thanks – applied.